driver/src/clawfc/Driver.java

```diff
@@ -0,0 +1,1545 @@
+                Path oldPath = fileNames.put(filename, inFilePath);
+                if (oldPath != null)
+                {
+                    throw new Exception(sprintf("Input files cannot have identical names: \n\"%s\"\n\"%s\"",
+                            oldPath.toString(), inFilePath.toString()));
+                }
+            }
+
+        }
+        if (opts.fortranCompilerType() != null ^ opts.fortranCompilerCmd() != null)
+        {
+            throw new Exception(sprintf("Options --fc-type and --fc-cmd must be specified together"));
+        }
+        for (Path incPath : opts.preprocessorIncludeDirs())
+        {
+            if (!Utils.dirExists(incPath))
+            {
+                throw new Exception(
+                        sprintf("Include dir \"%s\" does not exist or is not a directory", incPath.toString()));
+            }
+        }
+        for (Path incPath : opts.buildInfoIncludeDirs())
+        {
+            if (!Utils.dirExists(incPath))
+            {
+                throw new Exception(sprintf("Buildinfo include dir \"%s\" does not exist or is not a directory",
+                        incPath.toString()));
+            }
+        }
+        for (Path incPath : opts.moduleIncludeDirs())
+        {
+            if (!Utils.dirExists(incPath))
+            {
+                throw new Exception(
+                        sprintf("Module include dir \"%s\" does not exist or is not a directory", incPath.toString()));
+            }
+        }
+        if (!opts.inputFiles().isEmpty())
+        {
+            if (opts.stopAfterPreprocessing())
+            {
+                return;
+            }
+            if (opts.buildInfoOutputDir() != null)
+            {
+                if (opts.preprocessedSourcesOutputDir() == null && !opts.skipPreprocessing())
+                {
+                    throw new Exception("Output directory for preprocessed FORTRAN source files -PO/--pp-output-dir"
+                            + " must be specified too when using -BO/--buildinfo-output-dir");
+                }
+            }
+            if (opts.stopAfterDepScan() || opts.generateBuildInfoFiles() || opts.printCLAWFiles())
+            {
+                if (opts.generateBuildInfoFiles())
+                {
+                    if (opts.buildInfoOutputDir() == null)
+                    {
+                        throw new Exception("Output buildinfo dir must be specified when using --gen-buildinfo-files");
+                    }
+                }
+                return;
+            }
+            if (opts.generateModFiles())
+            {
+                if (opts.xmodOutputDir() == null)
+                {
+                    throw new Exception("Output directory for .xmod files must be specified");
+                }
+                return;
+            }
+            if (opts.stopAfterXmodGeneration())
+            {
+            } else if (opts.stopAfterXastGeneration())
```

Comment:
Remove  empty else if

Reply:
It's like this on purpose, helps during debugging. As long as there's a line with opening bracket, I can put a breakpoint there.

Reply:
Conditional breakpoint should work fine for this purpose.